app/routes/auth/signin-oidc.js

```diff
@@ -81,7 +81,7 @@ module.exports = [{
             cphError: err instanceof NoEligibleCphError,
             hasMultipleBusineses: attachedToMultipleBusinesses,
             backLink: auth.requestAuthorizationCodeUrl(session, request),
-            sbi: organisation?.sbi,
+            sbiText: organisation?.sbi !== undefined ? ` - SBI ${organisation.sbi}` : null,
```

Comment:
null may work equally well, I don't know but would it be safer to use '' (empty string) here?

Reply:
typeof organisation?.sbi !== undefined

Reply:
"If a value is undefined or null, nothing is displayed. The same behavior occurs when referencing undefined or null objects. The following all output nothing if foo is undefined: {{ foo }}, {{ foo.bar }}, {{ foo.bar.baz }}."

https://mozilla.github.io/nunjucks/templating.html#:~:text=If%20a%20value%20is%20undefined%20or%20null%2C%20nothing%20is%20displayed.%20The%20same%20behavior%20occurs%20when%20referencing%20undefined%20or%20null%20objects.%20The%20following%20all%20output%20nothing%20if%20foo%20is%20undefined%3A%20%7B%7B%20foo%20%7D%7D%2C%20%7B%7B%20foo.bar%20%7D%7D%2C%20%7B%7B%20foo.bar.baz%20%7D%7D.